src/Template/Layout/js/app/components/filter-box.js

```diff
@@ -73,6 +73,15 @@ export default {
                         </select>
                     </span>
 
+                    <span v-else-if="filter.type === 'checkbox'">
+                        <label>Both</label>
+                        <input type="radio" v-model="queryFilter.filter[filter.name]" value="">
+                        <label>Yes</label>
+                        <input type="radio" v-model="queryFilter.filter[filter.name]" value="true">
+                        <label>No</label>
+                        <input type="radio" v-model="queryFilter.filter[filter.name]" value="false">
```

Comment:
why not use a `select` instead of `radio` ?

Reply:
that was @xho request